gputop/remote/gputop.js

```diff
@@ -95,14 +94,14 @@ if (!is_nodejs) {
 
     http_get('index.html', function (index_html) {
         var jsdom = require('jsdom');
-        jsdom.env({ html: index_data,
+        jsdom.env({ html: index_html,
```

Comment:
I think it could be good to see if we can avoid fetching index.html for node.js based tools - though maybe as a follow on patch if there's some reason that's awkward currently. conceptually it doesn't seem like we should need it.